ElementCreator.cs

```diff
@@ -1,40 +1,284 @@
-﻿using System;
+﻿using Microsoft.Maui.Controls.Shapes;
+using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Threading.Tasks;
+using Microsoft.Maui.Controls;
 
 namespace MAUIDesigner
 {
-    internal static class ElementCreator
+    internal abstract class ElementFactory
```

Comment:
This shouldn't be done. Here we are creating elements using reflection, so any new elements that are added to the Framework shouldn't need to be updated here. 
Instead let's create the factory methods for only those elements that are not getting displayed, like Shapes.